frontend/packages/pydantic-forms/src/core/helper.ts

```diff
@@ -194,49 +204,56 @@ export const getFlatFieldMap = (
     return fieldMap;
 };
 
+export const isNullable = (schema: PydanticFormPropertySchemaParsed) => {
```

Comment:
Those JS doc descriptions could be really useful. Can we start adding them for this sort of functions similar to the one below.